src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1052 @@ metadata_pkg.(
+# ridge_mach = machine(ridge_regressor, X, y) |> fit!
+# coef = fitted_params(ridge_mach).coefficients
+# difference = llsq_coef - coef
+# @info "difference between λ=0 ridge and llsq" mean(difference) std(difference)
+
+
+ridge_regressor = RidgeRegressor(lambda=1.5)
+ridge_mach = machine(ridge_regressor, X, y) |> fit!
+
+Xnew, _ = make_regression(3, 60)
+yhat = predict(mach, Xnew) # new predictions
+```
+
+See also
+[`LinearRegressor`](@ref), [`MultitargetLinearRegressor`](@ref), [`RidgeRegressor`](@ref)
+"""
+MultitargetRidgeRegressor
+
+"""
+$(MMI.doc_header(PCA))
+
+`PCA` Principal component analysis. Learns a linear transformation to
+project the data  on a lower dimensional space while preserving most of the initial
+variance.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `schema(X)`
+
+# Hyper-parameters
+
+- `maxoutdim=0`: The maximum number of output dimensions. If not set, defaults to
+  0, where all components are kept (e.g., the number of components/output dimensions
+  is equal to the size of the smallest dimension of the training matrix)
+- `method=:auto`: The method to use to solve the problem. Choices are
+    - `:svd`: Support Vector Decomposition of the matrix.
+    - `:cov`: Covariance matrix decomposition.
+    - `:auto`: Use `:cov` if the matrices first dimension is smaller than its second dimension
+      otherwise use `:svd`
+- `pratio::Float64=0.99`: The ratio of variance preserved after the transformation
+- `mean=nothing`: if set to nothing(default) centering will be computed and applied,
+  if set to `0` no centering(assumed pre-centered), if a vector is passed,
+  the centering is done with that vector.
+
+# Operations
+
+- `transform(mach, Xnew)`: Return lower dimensional projection of the target given new
+  features `Xnew` having the same Scitype as `X` above.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `projection`: Returns the projection matrix (of size `(d, p)`).
+  Each column of the projection matrix corresponds to a principal component.
+  The principal components are arranged in descending order of
+  the corresponding variances.
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `indim`: Dimensions of the provided data.
+- `outdim`: Dimensions of the transformed result.
+- `tprincipalvar`: Total variance of the principal components.
+- `tresidualvar`: Total residual variance.
+- `tvar`: Total observation variance (principal + residual variance).
+- `mean`: The mean vector (of length `d`).
+- `principalvars`: The variance of the principal components.
+
+# Examples
+
+```
+using MLJ
+
+PCA = @load PCA pkg=MultivariateStats
+
+X, y = @load_iris
+
+model = PCA(maxoutdim=2)
+mach = machine(model, X) |> fit!
+
+projection = transform(mach, X)
```

Comment:
Maybe its the mathematician speaking, but I tend to think of "projection" as a map, rather than an image under that map. How about "Xsmall" or "Xreduced" instead of "projection". 

As suggested above, lets compute `inverse_transform` of "Xsmall" and compare to the original "X" .